app/routes/circulars/classic.mdx

```diff
@@ -11,6 +11,15 @@ import {
 
 import { validSubjectKeywords } from './circulars.lib'
 
+<div className="usa-alert usa-alert--warning">
+  <div className="usa-alert__body">
+    <p className="usa-alert__text">
+      This page will be updated on April 17, 2023. See [news and
+      announcements](/news) for more information.
+    </p>
+  </div>
+</div>
```

Comment:
Use the react-uswds Alert component here.

Reply:
Updated in latest commit